deterministic-build.yml

```
@@ -0,0 +1,36 @@
+trigger:
+- master
+
+pool:
+  vmImage: windows-latest
+
+steps:
+- checkout: self
+  path: WalletWasabi
+
+- task: UseDotNet@2
+  displayName: 'Install .NET Core 3'
+  inputs:
+    version: 3.1.x
+
+- task: PowerShell@2
+  displayName: 'Build and compare'
+  inputs:
+    targetType: 'inline'
+    script: |
+      # Change git output from stderr to stdout https://github.com/dahlbyk/posh-git/issues/109.
+      $env:GIT_REDIRECT_STDERR = '2>&1'
+      cd ..
+      robocopy WalletWasabi WalletWasabiClone /COPYALL /S /XD ".git" /NFL /NDL /NJH
```

I don't think that anyone can understand this without a comment. It should be documented to give a reader a clue what is going on here.

```suggestion
      # Copies all file information recursively (expect empty directories), exclude `.git` subdirectories, do not log file names & directories.
      robocopy WalletWasabi WalletWasabiClone /COPYALL /S /XD ".git" /NFL /NDL /NJH
```